x-pack/test/functional/apps/rollup_job/hybrid_index_pattern.js

```diff
@@ -91,10 +90,6 @@ export default function ({ getService, getPageObjects }) {
       );
       expect(filteredIndexPatternNames.length).to.be(1);
 
-      // make sure there are no toasts which might be showing unexpected errors
```

Comment:
nit: not sure if this is an expected change.
Original code doesn't look good to me because I think it waits for a toast for a long time making the test very slow? But I think the check was there for a reason.

Reply:
This was a purposeful change. I'm not aware of the case where the index pattern creation could succeed (there are a number of checks for this) and yet display a toast error.